lib/train/file/remote/unix.rb

```diff
@@ -22,8 +22,13 @@ def content
         def exist?
           @exist ||= begin
             f = @follow_symlink ? "" : " || test -L #{@spath}"
-            @backend.run_command("test -e #{@spath}" + f)
-              .exit_status == 0
+            if @backend.run_command("uname -s").stdout.chomp == "SunOS"
+              # Solaris does not support `-e` flag:
```

Comment:
This is confusing because we're still using -e flag. It should explain that the default `test` does not so we specify running `/usr/bin/test` instead

Reply:
Really good point +1 thanks!